sasmodels/compare.py

```diff
@@ -1214,32 +1214,56 @@ def parse_opts(argv):
         'ngauss'    : '0',
     }
     for arg in flags:
-        if arg == '-noplot':    opts['plot'] = False
-        elif arg == '-plot':    opts['plot'] = True
-        elif arg == '-linear':  opts['view'] = 'linear'
-        elif arg == '-log':     opts['view'] = 'log'
-        elif arg == '-q4':      opts['view'] = 'q4'
-        elif arg == '-1d':      opts['is2d'] = False
-        elif arg == '-2d':      opts['is2d'] = True
-        elif arg == '-exq':     opts['qmax'] = 10.0
-        elif arg == '-highq':   opts['qmax'] = 1.0
-        elif arg == '-midq':    opts['qmax'] = 0.2
-        elif arg == '-lowq':    opts['qmax'] = 0.05
-        elif arg == '-sesans':  opts['sesans'] = True
-        elif arg == '-zero':    opts['zero'] = True
-        elif arg.startswith('-nq='):       opts['nq'] = int(arg[4:])
```

Comment:
I marked the wrong block with the PEP 8 consistency comment. For this very large switch statement I value vertical compactness over strict adherence to formatting conventions.